account_financial_report/__openerp__.py

```diff
@@ -39,6 +39,7 @@
         "security/security.xml",
         "security/ir.model.access.csv",
         "data/report_paperformat.xml",
+        "demo/demo.xml",
```

Comment:
Following the [file-naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) change: `s/demo.xml/account_demo.xml/g`